lib/spdy-transport/protocol/http2/constants.js

```diff
@@ -17,6 +17,7 @@ exports.INITIAL_MAX_FRAME_SIZE = 16384;
 exports.ABSOLUTE_MAX_FRAME_SIZE = 16777215;
 exports.HEADER_TABLE_SIZE = 4096;
 exports.DEFAULT_MAX_HEADER_LIST_SIZE = 80 * 1024;  // as in http_parser
+exports.MAX_INITIAL_WINDOW_SIZE = 2147483647;
```

Comment:
This should be duplicated in `spdy/constants.js` if you wish to use it in `connection.js`